inc/inventoryrulelocationcollection.class.php

```diff
@@ -95,10 +95,6 @@ function getTitle() {
     * @return array
     */
    function prepareInputDataForProcess($input, $params) {
```

Comment:
if there's nothing in this method now, why not to remove it ?

Reply:
Ok